sites/example-project/src/components/ui/QueryViewerSupport/QueryDataTable.svelte

```diff
@@ -83,12 +83,21 @@
       dataPage = updatedSlice
     }
 
+    function handleWheel(event){
+      // abort if scroll is in x-direction
+      if (Math.abs(event.deltaX) > Math.abs(event.deltaY)) {
+        return
+      }
+
+      event.preventDefault()
+      index= Math.min(Math.max(0, index + Math.floor(event.deltaY/Math.abs(event.deltaY))), max)
+      slice()
+    }
 </script>
 
 <div class="results-pane" transition:slide|local>
 <div class="container" >
-
-  <table in:blur>
+  <table in:blur on:wheel={handleWheel}>
```

Comment:
I think it should probably only scroll when the users is over the body of the table, not the header.
probably just a case of moving the on:wheel into the `<tbody>`

Reply:
Addressed